src/main/resources/native/ios/AppDelegate.m

```diff
@@ -53,7 +53,8 @@ @interface AppDelegate : UIResponder <UIApplicationDelegate>
 
 int startGVM();
 
-extern void *IsolateEnterStub__JavaMainWrapper__run__5087f5482cc9a6abc971913ece43acb471d2631b__a61fe6c26e84dd4037e4629852b5488bfcc16e7e();
+extern int *run_main(int argc, char* argv[]);
+const char * args[] = {"myapp"};
```

Comment:
Why `my app`? Can we use any name? Maybe we could replace the name we already have in ProjectConfiguration?

Reply:
a follow-up issue will allow to pass JVM parameters, including appname. But we first need to get all platforms on the same level (being able to pass a *args[])

Reply:
The reason for not doing it here is that it is not trivial to pass a value from a pom.xml into a c file that needs to be compiled at build time.